<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassGenerator</class>
 <widget class="QWidget" name="ClassGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClassGenerator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Class name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditClassName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxDisableCopy">
     <property name="text">
      <string>disable copy and assignment operator</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxUsePimpl">
     <property name="text">
      <string>use pImpl</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>generate in folder:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxFolder">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSelectFolder">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClearHistory">
       <property name="text">
        <string>clear history</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonStart">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
